modules/drivers/druid/src/metabase/driver/druid.clj

```diff
@@ -13,8 +13,9 @@
 
 (driver/register! :druid)
 
-(doseq [[feature supported?] {:set-timezone            true
-                              :expression-aggregations true}]
+(doseq [[feature supported?] {:expression-aggregations true
+                              :schemas                 false
```

Comment:
This was a previous bug with druid that didn't have any impact before this change, because previously the `:schemas` feature was only being used for uploads. Now it's being used for sync. The good news is I introduced the `:schemas` feature 3 releases ago in 47, which means 3rd party drivers will have had 3 releases notice to support it. 

They shouldn't have ignored it because I added a test for it [here](https://github.com/metabase/metabase/pull/41588/files#diff-a1b51f96feece082d660d8d97b049f2ebad66dfb2de5a095ac8ff848fac59e38R67) in 47.